packages/frontend/src/components/PinnedItemsPanel/index.tsx

```diff
@@ -55,6 +60,40 @@ const PinnedItemsPanel: FC<Props> = ({ projectUuid, organizationUuid }) => {
                     : 'Your data team have pinned these items to help guide you towards the most relevant content!',
             }}
         />
+    ) : userCanUpdateProject && pinnedItems.length <= 0 ? (
+        // FIXME: update width with Mantine widths
+        <Card
+            w={900}
```

Comment:
do we need to use `w={900}` here? or can we just say `w="100%"`

Reply:
removed it completely, card inherits parent width 👍🏼